pkg/checkpoint/cri/README.md

```diff
@@ -7,4 +7,4 @@ We copy generated CRI APIs for a couple reasons:
 The various versions of CRI are taken from:
 
 * v1alpha1: https://github.com/kubernetes/kubernetes/tree/v1.9.6/pkg/kubelet/apis/cri
-* v1alpha2: https://github.com/kubernetes/kubernetes/tree/v1.10.0-rc.1/pkg/kubelet/apis/cri
+* v1alpha2: https://github.com/kubernetes/cri-api/tree/release-1.16/pkg/apis
```

Comment:
IMHO its safer to be explicit about the version since, in theory, we could build two different binaries (e.g. one with 1.16.1, and another with 1.16.2) on two different days without ever changes the code. I propose we use `https://github.com/kubernetes/cri-api/tree/kubernetes-1.16.2`

Reply:
Ill change the readme, the branches are the same, presently, does that work ?

Reply:
For sure, if `release-1.16` was `kubernetes-1.16.2` when you pulled these in then 👍. I think in the long run we will import go code instead of copying the protobuf definitions, so this should be fine for now.